### DISCUSSION TITLE
vsgdynamicload.cpp     use the model's own location

### DISCUSSION THREAD

**Discussion by WdsPjm** [vsg-dev/VulkanSceneGraph#886] [Q&A] [answered]
In the vsgdynamicload.cpp demo, after I load the model, I want to use the model's own location. How do I operate

**Comment by robertosfield** [comment, marked as answer]
It's just a an example that loads lots of unrelated models and to do this has to translate and scale them to fit into the same space.

You could remove the vsg::MatrixTransform that is used to do the translate and scale and then would end up in their own coordinate frame but would make the example less flexible at handling unrelated models.

It think you find it more helpful explaining what you want to achieve rather than focusing on one specific example that is meant to just demonstrate dynamic loading.
